constraint/pkg/client/client.go

```diff
@@ -584,10 +586,17 @@ func (c *Client) review(ctx context.Context, target string, constraints []*unstr
 		tracesBuilder.WriteString("\n\n")
 	}
 
+	var statsEntries []*instrumentation.StatsEntry
+	if localDriver, ok := c.driver.(*local.Driver); ok {
```

Comment:
if we're fine w this approach, we could make the `local.Driver` have a func of its own. I'd just be weary about modifying the interface w the current approach (of storing stats in a variable) to not back ourselves in a corner. That is, other drivers can have a different style of storing and handling `StatsEntries`